### ISSUE TITLE
[1.7.10]: TrowingSpeers 

### ISSUE THREAD

**Issue by GamebP** [quentin452/1.7.10-Forge-renewed-v1#903]
### Modpack

Bagges Pack 1.7.10

### Game log

```java
[20:50:27] [Client thread/TRACE] [throwingspears/throwingspears]: Sending event FMLInitializationEvent to mod throwingspears
[20:50:27] [Client thread/INFO] [STDERR/throwingspears]: [java.lang.Throwable$WrappedPrintStream:println:749]: java.io.FileNotFoundException: https://dl.dropboxusercontent.com/u/37637724/MinecraftModVersions/throwingspears.version
[20:50:27] [Client thread/INFO] [STDERR/throwingspears]: [java.lang.Throwable$WrappedPrintStream:println:749]: 	at sun.net.www.protocol.http.HttpURLConnection.getInputStream0(HttpURLConnection.java:1898)
[20:50:27] [Client thread/INFO] [STDERR/throwingspears]: [java.lang.Throwable$WrappedPrintStream:println:749]: 	at sun.net.www.protocol.http.HttpURLConnection.access$200(HttpURLConnection.java:92)
[20:50:27] [Client thread/INFO] [STDERR/throwingspears]: [java.lang.Throwable$WrappedPrintStream:println:749]: 	at sun.net.www.protocol.http.HttpURLConnection$9.run(HttpURLConnection.java:1492)
[20:50:27] [Client thread/INFO] [STDERR/throwingspears]: [java.lang.Throwable$WrappedPrintStream:println:749]: 	at sun.net.www.protocol.http.HttpURLConnection$9.run(HttpURLConnection.java:1490)
[20:50:27] [Client thread/INFO] [STDERR/throwingspears]: [java.lang.Throwable$WrappedPrintStream:println:749]: 	at java.security.AccessController.doPrivileged(Native Method)
[20:50:27] [Client thread/INFO] [STDERR/throwingspears]: [java.lang.Throwable$WrappedPrintStream:println:749]: 	at java.security.AccessController.doPrivilegedWithCombiner(AccessController.java:784)
[20:50:27] [Client thread/INFO] [STDERR/throwingspears]: [java.lang.Throwable$WrappedPrintStream:println:749]: 	at sun.net.www.protocol.http.HttpURLConnection.getInputStream(HttpURLConnection.java:1489)
[20:50:27] [Client thread/INFO] [STDERR/throwingspears]: [java.lang.Throwable$WrappedPrintStream:println:749]: 	at sun.net.www.protocol.https.HttpsURLConnectionImpl.getInputStream(HttpsURLConnectionImpl.java:268)
[20:50:27] [Client thread/INFO] [STDERR/throwingspears]: [java.lang.Throwable$WrappedPrintStream:println:749]: 	at java.net.URL.openStream(URL.java:1093)
[20:50:27] [Client thread/INFO] [STDERR/throwingspears]: [java.lang.Throwable$WrappedPrintStream:println:749]: 	at grim3212.mc.core.util.VersionChecker.<init>(VersionChecker.java:22)
[20:50:27] [Client thread/INFO] [STDERR/throwingspears]: [java.lang.Throwable$WrappedPrintStream:println:749]: 	at grim3212.mc.core.GrimModule.registerVersionCheck(GrimModule.java:63)
[20:50:27] [Client thread/INFO] [STDERR/throwingspears]: [java.lang.Throwable$WrappedPrintStream:println:749]: 	at grim3212.mc.core.GrimModule.registerMod(GrimModule.java:43)
[20:50:27] [Client thread/INFO] [STDERR/throwingspears]: [java.lang.Throwable$WrappedPrintStream:println:749]: 	at grim3212.mc.throwingspears.ThrowingSpears.load(ThrowingSpears.java:111)
[20:50:27] [Client thread/INFO] [STDERR/throwingspears]: [java.lang.Throwable$WrappedPrintStream:println:749]: 	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
[20:50:27] [Client thread/INFO] [STDERR/throwingspears]: [java.lang.Throwable$WrappedPrintStream:println:749]: 	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
[20:50:27] [Client thread/INFO] [STDERR/throwingspears]: [java.lang.Throwable$WrappedPrintStream:println:749]: 	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
[20:50:27] [Client thread/INFO] [STDERR/throwingspears]: [java.lang.Throwable$WrappedPrintStream:println:749]: 	at java.lang.reflect.Method.invoke(Method.java:498)
[20:50:27] [Client thread/INFO] [STDERR/throwingspears]: [java.lang.Throwable$WrappedPrintStream:println:749]: 	at cpw.mods.fml.common.FMLModContainer.handleModStateEvent(FMLModContainer.java:532)
[20:50:27] [Client thread/INFO] [STDERR/throwingspears]: [java.lang.Throwable$WrappedPrintStream:println:749]: 	at sun.reflect.GeneratedMethodAccessor12.invoke(Unknown Source)
[20:50:27] [Client thread/INFO] [STDERR/throwingspears]: [java.lang.Throwable$WrappedPrintStream:println:749]: 	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
[20:50:27] [Client thread/INFO] [STDERR/throwingspears]: [java.lang.Throwable$WrappedPrintStream:println:749]: 	at java.lang.reflect.Method.invoke(Method.java:498)
[20:50:27] [Client thread/INFO] [STDERR/throwingspears]: [java.lang.Throwable$WrappedPrintStream:println:749]: 	at com.google.common.eventbus.EventSubscriber.handleEvent(EventSubscriber.java:74)
[20:50:27] [Client thread/INFO] [STDERR/throwingspears]: [java.lang.Throwable$WrappedPrintStream:println:749]: 	at com.google.common.eventbus.SynchronizedEventSubscriber.handleEvent(SynchronizedEventSubscriber.java:47)
[20:50:27] [Client thread/INFO] [STDERR/throwingspears]: [java.lang.Throwable$WrappedPrintStream:println:749]: 	at com.google.common.eventbus.EventBus.dispatch(EventBus.java:322)
[20:50:27] [Client thread/INFO] [STDERR/throwingspears]: [java.lang.Throwable$WrappedPrintStream:println:749]: 	at com.google.common.eventbus.EventBus.dispatchQueuedEvents(EventBus.java:304)
[20:50:27] [Client thread/INFO] [STDERR/throwingspears]: [java.lang.Throwable$WrappedPrintStream:println:749]: 	at com.google.common.eventbus.EventBus.post(EventBus.java:275)
[20:50:27] [Client thread/INFO] [STDERR/throwingspears]: [java.lang.Throwable$WrappedPrintStream:println:749]: 	at cpw.mods.fml.common.LoadController.sendEventToModContainer(LoadController.java:212)
[20:50:27] [Client thread/INFO] [STDERR/throwingspears]: [java.lang.Throwable$WrappedPrintStream:println:749]: 	at cpw.mods.fml.common.LoadController.propogateStateMessage(LoadController.java:190)
[20:50:27] [Client thread/INFO] [STDERR/throwingspears]: [java.lang.Throwable$WrappedPrintStream:println:749]: 	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
[20:50:27] [Client thread/INFO] [STDERR/throwingspears]: [java.lang.Throwable$WrappedPrintStream:println:749]: 	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
[20:50:27] [Client thread/INFO] [STDERR/throwingspears]: [java.lang.Throwable$WrappedPrintStream:println:749]: 	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
[20:50:27] [Client thread/INFO] [STDERR/throwingspears]: [java.lang.Throwable$WrappedPrintStream:println:749]: 	at java.lang.reflect.Method.invoke(Method.java:498)
[20:50:27] [Client thread/INFO] [STDERR/throwingspears]: [java.lang.Throwable$WrappedPrintStream:println:749]: 	at com.google.common.eventbus.EventSubscriber.handleEvent(EventSubscriber.java:74)
[20:50:27] [Client thread/INFO] [STDERR/throwingspears]: [java.lang.Throwable$WrappedPrintStream:println:749]: 	at com.google.common.eventbus.SynchronizedEventSubscriber.handleEvent(SynchronizedEventSubscriber.java:47)
[20:50:27] [Client thread/INFO] [STDERR/throwingspears]: [java.lang.Throwable$WrappedPrintStream:println:749]: 	at com.google.common.eventbus.EventBus.dispatch(EventBus.java:322)
[20:50:27] [Client thread/INFO] [STDERR/throwingspears]: [java.lang.Throwable$WrappedPrintStream:println:749]: 	at com.google.common.eventbus.EventBus.dispatchQueuedEvents(EventBus.java:304)
[20:50:27] [Client thread/INFO] [STDERR/throwingspears]: [java.lang.Throwable$WrappedPrintStream:println:749]: 	at com.google.common.eventbus.EventBus.post(EventBus.java:275)
[20:50:27] [Client thread/INFO] [STDERR/throwingspears]: [java.lang.Throwable$WrappedPrintStream:println:749]: 	at cpw.mods.fml.common.LoadController.distributeStateMessage(LoadController.java:119)
[20:50:27] [Client thread/INFO] [STDERR/throwingspears]: [java.lang.Throwable$WrappedPrintStream:println:749]: 	at cpw.mods.fml.common.Loader.initializeMods(Loader.java:737)
[20:50:27] [Client thread/INFO] [STDERR/throwingspears]: [java.lang.Throwable$WrappedPrintStream:println:749]: 	at cpw.mods.fml.client.FMLClientHandler.finishMinecraftLoading(FMLClientHandler.java:311)
[20:50:27] [Client thread/INFO] [STDERR/throwingspears]: [java.lang.Throwable$WrappedPrintStream:println:749]: 	at net.minecraft.client.Minecraft.func_71384_a(Minecraft.java:552)
[20:50:27] [Client thread/INFO] [STDERR/throwingspears]: [java.lang.Throwable$WrappedPrintStream:println:749]: 	at net.minecraft.client.Minecraft.func_99999_d(Minecraft.java:8993)
[20:50:27] [Client thread/INFO] [STDERR/throwingspears]: [java.lang.Throwable$WrappedPrintStream:println:749]: 	at net.minecraft.client.main.Main.main(SourceFile:148)
[20:50:27] [Client thread/INFO] [STDERR/throwingspears]: [java.lang.Throwable$WrappedPrintStream:println:749]: 	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
[20:50:27] [Client thread/INFO] [STDERR/throwingspears]: [java.lang.Throwable$WrappedPrintStream:println:749]: 	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
[20:50:27] [Client thread/INFO] [STDERR/throwingspears]: [java.lang.Throwable$WrappedPrintStream:println:749]: 	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
[20:50:27] [Client thread/INFO] [STDERR/throwingspears]: [java.lang.Throwable$WrappedPrintStream:println:749]: 	at java.lang.reflect.Method.invoke(Method.java:498)
[20:50:27] [Client thread/INFO] [STDERR/throwingspears]: [java.lang.Throwable$WrappedPrintStream:println:749]: 	at net.minecraft.launchwrapper.Launch.launch(Launch.java:135)
[20:50:27] [Client thread/INFO] [STDERR/throwingspears]: [java.lang.Throwable$WrappedPrintStream:println:749]: 	at net.minecraft.launchwrapper.Launch.main(Launch.java:28)
[20:50:27] [Client thread/INFO] [STDERR/throwingspears]: [java.lang.Throwable$WrappedPrintStream:println:749]: 	at org.prismlauncher.launcher.impl.StandardLauncher.launch(StandardLauncher.java:88)
[20:50:27] [Client thread/INFO] [STDERR/throwingspears]: [java.lang.Throwable$WrappedPrintStream:println:749]: 	at org.prismlauncher.EntryPoint.listen(EntryPoint.java:126)
[20:50:27] [Client thread/INFO] [STDERR/throwingspears]: [java.lang.Throwable$WrappedPrintStream:println:749]: 	at org.prismlauncher.EntryPoint.main(EntryPoint.java:71)
[20:50:27] [Client thread/ERROR] [Grim3212 Core/throwingspears]: Error retrieving version check data for throwingspears
[20:50:27] [Client thread/TRACE] [throwingspears/throwingspears]: Sent event FMLInitializationEvent to mod throwingspears
```

### Description

issue when downloading a closed file that's not uploaded anymore

**Comment by GamebP** [comment]
Same as wallclock
```java
[20:50:30] [Client thread/TRACE] [wallclock/wallclock]: Sending event FMLInitializationEvent to mod wallclock
[20:50:31] [Client thread/INFO] [STDERR/wallclock]: [java.lang.Throwable$WrappedPrintStream:println:749]: java.io.FileNotFoundException: https://dl.dropboxusercontent.com/u/37637724/MinecraftModVersions/wallclock.version
[20:50:31] [Client thread/INFO] [STDERR/wallclock]: [java.lang.Throwable$WrappedPrintStream:println:749]: 	at sun.net.www.protocol.http.HttpURLConnection.getInputStream0(HttpURLConnection.java:1898)
[20:50:31] [Client thread/INFO] [STDERR/wallclock]: [java.lang.Throwable$WrappedPrintStream:println:749]: 	at sun.net.www.protocol.http.HttpURLConnection.access$200(HttpURLConnection.java:92)
[20:50:31] [Client thread/INFO] [STDERR/wallclock]: [java.lang.Throwable$WrappedPrintStream:println:749]: 	at sun.net.www.protocol.http.HttpURLConnection$9.run(HttpURLConnection.java:1492)
[20:50:31] [Client thread/INFO] [STDERR/wallclock]: [java.lang.Throwable$WrappedPrintStream:println:749]: 	at sun.net.www.protocol.http.HttpURLConnection$9.run(HttpURLConnection.java:1490)
[20:50:31] [Client thread/INFO] [STDERR/wallclock]: [java.lang.Throwable$WrappedPrintStream:println:749]: 	at java.security.AccessController.doPrivileged(Native Method)
[20:50:31] [Client thread/INFO] [STDERR/wallclock]: [java.lang.Throwable$WrappedPrintStream:println:749]: 	at java.security.AccessController.doPrivilegedWithCombiner(AccessController.java:784)
[20:50:31] [Client thread/INFO] [STDERR/wallclock]: [java.lang.Throwable$WrappedPrintStream:println:749]: 	at sun.net.www.protocol.http.HttpURLConnection.getInputStream(HttpURLConnection.java:1489)
[20:50:31] [Client thread/INFO] [STDERR/wallclock]: [java.lang.Throwable$WrappedPrintStream:println:749]: 	at sun.net.www.protocol.https.HttpsURLConnectionImpl.getInputStream(HttpsURLConnectionImpl.java:268)
[20:50:31] [Client thread/INFO] [STDERR/wallclock]: [java.lang.Throwable$WrappedPrintStream:println:749]: 	at java.net.URL.openStream(URL.java:1093)
[20:50:31] [Client thread/INFO] [STDERR/wallclock]: [java.lang.Throwable$WrappedPrintStream:println:749]: 	at grim3212.mc.core.util.VersionChecker.<init>(VersionChecker.java:22)
[20:50:31] [Client thread/INFO] [STDERR/wallclock]: [java.lang.Throwable$WrappedPrintStream:println:749]: 	at grim3212.mc.core.GrimModule.registerVersionCheck(GrimModule.java:63)
[20:50:31] [Client thread/INFO] [STDERR/wallclock]: [java.lang.Throwable$WrappedPrintStream:println:749]: 	at grim3212.mc.core.GrimModule.registerMod(GrimModule.java:43)
[20:50:31] [Client thread/INFO] [STDERR/wallclock]: [java.lang.Throwable$WrappedPrintStream:println:749]: 	at grim3212.mc.wallclock.WallClock.load(WallClock.java:70)
[20:50:31] [Client thread/INFO] [STDERR/wallclock]: [java.lang.Throwable$WrappedPrintStream:println:749]: 	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
[20:50:31] [Client thread/INFO] [STDERR/wallclock]: [java.lang.Throwable$WrappedPrintStream:println:749]: 	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
[20:50:31] [Client thread/INFO] [STDERR/wallclock]: [java.lang.Throwable$WrappedPrintStream:println:749]: 	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
[20:50:31] [Client thread/INFO] [STDERR/wallclock]: [java.lang.Throwable$WrappedPrintStream:println:749]: 	at java.lang.reflect.Method.invoke(Method.java:498)
[20:50:31] [Client thread/INFO] [STDERR/wallclock]: [java.lang.Throwable$WrappedPrintStream:println:749]: 	at cpw.mods.fml.common.FMLModContainer.handleModStateEvent(FMLModContainer.java:532)
[20:50:31] [Client thread/INFO] [STDERR/wallclock]: [java.lang.Throwable$WrappedPrintStream:println:749]: 	at sun.reflect.GeneratedMethodAccessor12.invoke(Unknown Source)
[20:50:31] [Client thread/INFO] [STDERR/wallclock]: [java.lang.Throwable$WrappedPrintStream:println:749]: 	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
[20:50:31] [Client thread/INFO] [STDERR/wallclock]: [java.lang.Throwable$WrappedPrintStream:println:749]: 	at java.lang.reflect.Method.invoke(Method.java:498)
[20:50:31] [Client thread/INFO] [STDERR/wallclock]: [java.lang.Throwable$WrappedPrintStream:println:749]: 	at com.google.common.eventbus.EventSubscriber.handleEvent(EventSubscriber.java:74)
[20:50:31] [Client thread/INFO] [STDERR/wallclock]: [java.lang.Throwable$WrappedPrintStream:println:749]: 	at com.google.common.eventbus.SynchronizedEventSubscriber.handleEvent(SynchronizedEventSubscriber.java:47)
[20:50:31] [Client thread/INFO] [STDERR/wallclock]: [java.lang.Throwable$WrappedPrintStream:println:749]: 	at com.google.common.eventbus.EventBus.dispatch(EventBus.java:322)
[20:50:31] [Client thread/INFO] [STDERR/wallclock]: [java.lang.Throwable$WrappedPrintStream:println:749]: 	at com.google.common.eventbus.EventBus.dispatchQueuedEvents(EventBus.java:304)
[20:50:31] [Client thread/INFO] [STDERR/wallclock]: [java.lang.Throwable$WrappedPrintStream:println:749]: 	at com.google.common.eventbus.EventBus.post(EventBus.java:275)
[20:50:31] [Client thread/INFO] [STDERR/wallclock]: [java.lang.Throwable$WrappedPrintStream:println:749]: 	at cpw.mods.fml.common.LoadController.sendEventToModContainer(LoadController.java:212)
[20:50:31] [Client thread/INFO] [STDERR/wallclock]: [java.lang.Throwable$WrappedPrintStream:println:749]: 	at cpw.mods.fml.common.LoadController.propogateStateMessage(LoadController.java:190)
[20:50:31] [Client thread/INFO] [STDERR/wallclock]: [java.lang.Throwable$WrappedPrintStream:println:749]: 	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
[20:50:31] [Client thread/INFO] [STDERR/wallclock]: [java.lang.Throwable$WrappedPrintStream:println:749]: 	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
[20:50:31] [Client thread/INFO] [STDERR/wallclock]: [java.lang.Throwable$WrappedPrintStream:println:749]: 	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
[20:50:31] [Client thread/INFO] [STDERR/wallclock]: [java.lang.Throwable$WrappedPrintStream:println:749]: 	at java.lang.reflect.Method.invoke(Method.java:498)
[20:50:31] [Client thread/INFO] [STDERR/wallclock]: [java.lang.Throwable$WrappedPrintStream:println:749]: 	at com.google.common.eventbus.EventSubscriber.handleEvent(EventSubscriber.java:74)
[20:50:31] [Client thread/INFO] [STDERR/wallclock]: [java.lang.Throwable$WrappedPrintStream:println:749]: 	at com.google.common.eventbus.SynchronizedEventSubscriber.handleEvent(SynchronizedEventSubscriber.java:47)
[20:50:31] [Client thread/INFO] [STDERR/wallclock]: [java.lang.Throwable$WrappedPrintStream:println:749]: 	at com.google.common.eventbus.EventBus.dispatch(EventBus.java:322)
[20:50:31] [Client thread/INFO] [STDERR/wallclock]: [java.lang.Throwable$WrappedPrintStream:println:749]: 	at com.google.common.eventbus.EventBus.dispatchQueuedEvents(EventBus.java:304)
[20:50:31] [Client thread/INFO] [STDERR/wallclock]: [java.lang.Throwable$WrappedPrintStream:println:749]: 	at com.google.common.eventbus.EventBus.post(EventBus.java:275)
[20:50:31] [Client thread/INFO] [STDERR/wallclock]: [java.lang.Throwable$WrappedPrintStream:println:749]: 	at cpw.mods.fml.common.LoadController.distributeStateMessage(LoadController.java:119)
[20:50:31] [Client thread/INFO] [STDERR/wallclock]: [java.lang.Throwable$WrappedPrintStream:println:749]: 	at cpw.mods.fml.common.Loader.initializeMods(Loader.java:737)
[20:50:31] [Client thread/INFO] [STDERR/wallclock]: [java.lang.Throwable$WrappedPrintStream:println:749]: 	at cpw.mods.fml.client.FMLClientHandler.finishMinecraftLoading(FMLClientHandler.java:311)
[20:50:31] [Client thread/INFO] [STDERR/wallclock]: [java.lang.Throwable$WrappedPrintStream:println:749]: 	at net.minecraft.client.Minecraft.func_71384_a(Minecraft.java:552)
[20:50:31] [Client thread/INFO] [STDERR/wallclock]: [java.lang.Throwable$WrappedPrintStream:println:749]: 	at net.minecraft.client.Minecraft.func_99999_d(Minecraft.java:8993)
[20:50:31] [Client thread/INFO] [STDERR/wallclock]: [java.lang.Throwable$WrappedPrintStream:println:749]: 	at net.minecraft.client.main.Main.main(SourceFile:148)
[20:50:31] [Client thread/INFO] [STDERR/wallclock]: [java.lang.Throwable$WrappedPrintStream:println:749]: 	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
[20:50:31] [Client thread/INFO] [STDERR/wallclock]: [java.lang.Throwable$WrappedPrintStream:println:749]: 	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
[20:50:31] [Client thread/INFO] [STDERR/wallclock]: [java.lang.Throwable$WrappedPrintStream:println:749]: 	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
[20:50:31] [Client thread/INFO] [STDERR/wallclock]: [java.lang.Throwable$WrappedPrintStream:println:749]: 	at java.lang.reflect.Method.invoke(Method.java:498)
[20:50:31] [Client thread/INFO] [STDERR/wallclock]: [java.lang.Throwable$WrappedPrintStream:println:749]: 	at net.minecraft.launchwrapper.Launch.launch(Launch.java:135)
[20:50:31] [Client thread/INFO] [STDERR/wallclock]: [java.lang.Throwable$WrappedPrintStream:println:749]: 	at net.minecraft.launchwrapper.Launch.main(Launch.java:28)
[20:50:31] [Client thread/INFO] [STDERR/wallclock]: [java.lang.Throwable$WrappedPrintStream:println:749]: 	at org.prismlauncher.launcher.impl.StandardLauncher.launch(StandardLauncher.java:88)
[20:50:31] [Client thread/INFO] [STDERR/wallclock]: [java.lang.Throwable$WrappedPrintStream:println:749]: 	at org.prismlauncher.EntryPoint.listen(EntryPoint.java:126)
[20:50:31] [Client thread/INFO] [STDERR/wallclock]: [java.lang.Throwable$WrappedPrintStream:println:749]: 	at org.prismlauncher.EntryPoint.main(EntryPoint.java:71)
[20:50:31] [Client thread/ERROR] [Grim3212 Core/wallclock]: Error retrieving version check data for wallclock```

**Comment by quentin452** [comment]
Fixed in V0.94.83 by updating OptimizationsAndTweaks by disabling Grim Version Checker
